gig-marketplace.md

```diff
@@ -0,0 +1,70 @@
+## Title 
+
+Gig Marketplace
+
+## Patlet  
+
+Establish a marketplace by creating an intranet website that lists specific InnerSource project needs as "Gigs" with explicit time and skill requirements.  This will enable managers to better understand their employee’s time commitment and professional benefits thereby increasing the likelihood of garnering approval to make InnerSource contributions.
+
+## Problem
+
+Neither managers nor employees understand how they might benefit from getting involved in an InnerSource project.
+
+It is difficult for employees to communicate to their management what commitment of time they will need to make to an InnerSource project.  
+
+Managers have no uniform way of keeping track of or rewarding their employee’s involvement in InnerSource projects.
+
+## Story
+ 
+You’ve successfully created an InnerSource program at your company and have buy-in from senior management, middle-management and developers.  However, after nearly a year there have been few actual contributions made to any InnerSource projects outside of the teams that originally created them.  After interviewing all the parties involved the main sticking point seems to be that it is difficult to know the time commitment developers will be asked to make if they choose to get involved in an InnerSource project and how they will personally benefit. There is also no uniform way of advertising what opportunities for contributors exist, what they’ll be asked to do and approximately how long it might take.  Managers are supportive and want their employees to participate but so far have lacked a way of accounting for or rewarding their employees’ activities within InnerSource projects.  What can be done to improve this situation for all the parties involved (InnerSource project owners, potential contributors and development managers)? 
+
+## Forces
+
+* Employees have no easy way to discover what InnerSource opportunities exist 
+* Employees don't understand how contributing might benefit them professionally
+* Managers don't understand the time/effort requirements associated with InnerSource project related tasks
+
+## Context
+
+Employees wish they could gain exposure to activities going on within other areas of the company without having to leave their current positions. InnerSource projects exist and could provide these experiences but there are two main factors preventing employees from participating. First, is the inability to easily discover what opportunities for contribution exist within ongoing InnerSource projects and to communicate these to their managers.  Second, is the inability of managers to plan and account for their employees' time commitments to these InnerSource project tasks.  As a result InnerSource project owners are finding it difficult to build communities of sufficient size to fulfill their stated goals.
+
+#### Prerequisites
```

Comment:
Minor: should this be a third level heading rather than fourth level?

Reply:
I'm personally not a fan of the way that third-level headings are rendered in _GitHub_ (I skip to 4th myself quite often).  Up to you in the end, though.

Reply:
That's fine.  I've updated this to be a 3rd level heading. Ready to go.